weather.go

```diff
@@ -25,26 +27,23 @@ type Weather struct {
 		Pressure  int     `json:"pressure"`
```

Comment:
Do we ever use `Pressure`? If not, we don't need to unmarshal it. You might know that `json.Unmarshal` ignores any JSON fields which aren't mentioned in the Go struct definition, so if we simply don't mention them here, they'll be skipped.